.github/compatibility-test-matrices/compat.py

```diff
@@ -2,78 +2,93 @@
 import argparse
 import os
 import json
+import enum
+from typing import Tuple, Generator, Optional, Dict, Any
 
 # JSON keys to search in.
-KEYS = ("chain-a", "chain-b")
+KEYS: Tuple[str, str] = ("chain-a", "chain-b")
 # Toggle if required. Indent = 4 matches jq.
-DUMP_ARGS = {"indent": 4, "sort_keys": False, "ensure_ascii": False}
+DUMP_ARGS: Dict[Any, Any] = {
```

Comment:
unfortunate but `my;py` throws a big hissy-fit here. `**kwargs` in general are a pain in typing iirc.